spec/Entity/LineItemSpec.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Sylius\InvoicingPlugin\Entity;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Resource\Model\ResourceInterface;
+use Sylius\InvoicingPlugin\Entity\InvoiceInterface;
+use Sylius\InvoicingPlugin\Entity\LineItemInterface;
+
+final class LineItemSpec extends ObjectBehavior
+{
+    function let(): void {
+        $this->beConstructedWith(
+            'Mjolnir',
+            2,
+            5000,
+            10000,
+            300,
+            10300
+        );
+    }
+
+    function it_implements_line_item_interface(): void
+    {
+        $this->shouldImplement(LineItemInterface::class);
+    }
+
+    function it_implements_resource_interface(): void
+    {
+        $this->shouldImplement(ResourceInterface::class);
+    }
+
+    function it_has_proper_line_item_data(): void
+    {
+        $this->beConstructedWith(
```

Comment:
Since you don't need to change constructor parameters in serveral tests, why don't you put this in let() function?